main/lxcmd.c

```diff
@@ -826,8 +828,10 @@ static boolean parseExtensionFields (tagEntry *const entry, char *const string,
 
 						}
 					}
-					else
-						; /* TODO: warning */
+					else {
+                                            /*TODO Handle warning*/
+                                            ;
+                                        }
```

Comment:
Style doesn't appear consistent here.  Should probably be either left as is (how is it harmful?) or follow common style, something like:
```C
else
{
	/* TODO: Handle warning */
}
```